src/scenes/game.py

```diff
@@ -64,6 +71,28 @@ def setup(self):
             self._screen.get_width() - open_in_game_menu_button.view.width - 10, 10)
         self._scene_objects.append(open_in_game_menu_button)
 
+        # for automation
+        self._script_callback = None
+        if self._script:
+            # pylint: disable=exec-used
```

Comment:
pylint complains about `exec`  being used, which in this implementation is necessary

Reply:
Could you move lines 77 to 94 to another method?